File: telegram/src/main/java/by/mrrockka/domain/PokerClockBotOptions.java
package by.mrrockka.domain;

import lombok.Builder;
import org.telegram.telegrambots.bots.DefaultBotOptions;

import java.util.List;

public class PokerClockBotOptions extends DefaultBotOptions {

  @Builder
  public PokerClockBotOptions(final List<String> allowedUpdates) {
    this.setAllowedUpdates(allowedUpdates);
  }
}

File: telegram/src/main/java/by/mrrockka/domain/PokerClockBotOptions.kt
package by.mrrockka.domain

import org.telegram.telegrambots.bots.DefaultBotOptions

@Deprecated("For removal")
data class PokerClockBotOptions(
        val updateTypes: List<String>,
) : DefaultBotOptions() {
    init {
        this.setAllowedUpdates(updateTypes)
    }
}
